cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -122,6 +122,7 @@ static const size_t nvidia_globs_len =
     sizeof nvidia_globs / sizeof *nvidia_globs;
 
 #define LIBNVIDIA_GLCORE_SO_PATTERN "libnvidia-glcore.so.%d.%d"
+#define LIBNVIDIA_GLCORE_SO_PATTERN_MICRO "libnvidia-glcore.so.%d.%d.%02d"
```

Comment:
`"libnvidia-glcore.so.%d.%d.%d"` should be fine too

Reply:
in my case the driver version is `440.33.01` so it should be `%02d`

Reply:
We should not assume the micro release number to be 2 digits only.

Reply:
well `printf('%02d', 420)` will print `420`, seems fine to me

Reply:
Sorry for providing confusing feedback. My mind is probably ready for the upcoming break already. Please see the other discussion https://github.com/snapcore/snapd/pull/7942#discussion_r360357409